5/README.md

```diff
@@ -0,0 +1,371 @@
+| plants | [StaticMenu.PlantsEntry](#opencannabis.products.menu.StaticMenu.PlantsEntry) | repeated | Specifies PLANT products attached to this menu. |
+| prerolls | [StaticMenu.PrerollsEntry](#opencannabis.products.menu.StaticMenu.PrerollsEntry) | repeated | Specifies PREROLL products attached to this menu. |
+
+
+<a name="opencannabis.products.menu.StaticMenu.ApothecaryEntry"/>
+
+### StaticMenu.ApothecaryEntry
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Apothecary](#opencannabis.products.Apothecary) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.CartridgesEntry"/>
+
+### StaticMenu.CartridgesEntry
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Cartridge](#opencannabis.products.Cartridge) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.EdiblesEntry"/>
+
+### StaticMenu.EdiblesEntry
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Edible](#opencannabis.products.Edible) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.ExtractsEntry"/>
+
+### StaticMenu.ExtractsEntry
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Extract](#opencannabis.products.Extract) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.FlowersEntry"/>
+
+### StaticMenu.FlowersEntry
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Flower](#opencannabis.products.Flower) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.MerchandiseEntry"/>
+
+### StaticMenu.MerchandiseEntry
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Merchandise](#opencannabis.products.Merchandise) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.PlantsEntry"/>
+
+### StaticMenu.PlantsEntry
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Plant](#opencannabis.products.Plant) |  |  |
+
+
+<a name="opencannabis.products.menu.StaticMenu.PrerollsEntry"/>
+
+### StaticMenu.PrerollsEntry
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| key | [string](#string) |  |  |
+| value | [opencannabis.products.Preroll](#opencannabis.products.Preroll) |  |  |
+
+
+<a name="opencannabis.products.menu.Flag"/>
+
+### Flag
+Enumerates flags that can be set on a menu.
+
```

Comment:
Specific Flags for a menu, I.E. if you have products you may not display so you have a "draft" version of your menu.